packages/@aws-cdk/aws-sns-subscriptions/test/integ.sns-lambda.ts

```diff
@@ -15,7 +15,20 @@ class SnsToSqs extends cdk.Stack {
       code: lambda.Code.inline(`exports.handler = ${handler.toString()}`)
     });
 
-    topic.addSubscription(new subs.LambdaSubscription(fction));
+    topic.addSubscription(new subs.LambdaSubscription(fction, {
```

Comment:
Let's add an additional test with this. It's usually not a good practice to piggy back on existing tests to extend coverage because then you lose the old case.